ontologies/works.ttl

```diff
@@ -0,0 +1,344 @@
+
+@prefix ww: <TBC/works> . 
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> . 
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix owl:     <http://www.w3.org/2002/07/owl#> .
+@prefix rdfs:    <http://www.w3.org/2000/01/rdf-schema#> .
+
+TBC
+    rdf:type owl:Ontology ;
+    dcterms:title "Works"@en ;
+    dcterms:created "13/02/2017"^^xsd:date ;
+    dcterms:description "Ontology describing the properties of works"@en .
+	
+
+#######  Classes ##### 	
+
+ww:Work rdf:type owl:Class ;
+	rdfs:label "Work"@en ;
+	rdfs:comment "An individual work such as a text, archive item or picture; or a grouping of individual works (so, for instance, an archive collection counts as a work, as do all the series and individual files within it).  Each work may exist in multiple instances (e.g. copies of the same book).  N.B. this is not synonymous with "work" as that is understood in FRBR but represents something lower down the FRBR hierarchy, namely manifestation. Groups of related items are also included as works because they have similar properties to the individual ones."@en ;
+	rdf:isDefinedBy <TBC/works> .
+	
+ww:Collection rdf:type owl:Class ;
+	rdfs:label "Collection"@en ;
+	rdfs:comment "A grouping of material comprising more than one work, the grouping being either organic (i.e., a Fonds) or artificial."@en ;
+	rdfs:subClassOf ww:Work
+	rdf:isDefinedBy <TBC/works> .
+	
+ww:Item rdf:type owl:Class ;
+	rdfs:label "Item"@en ;
+	rdfs:comment "A specific instance of a work: for instance, an individual physical copy with its own location.  This corresponds to the bottom, Instance, layer of FRBR.  Works that are unique - that is, have only one copy - are described in terms of a Work for their intellectual content, linked to an Item describing their phyiscal characteristics."@en ;
+	rdf:isDefinedBy <TBC/works> .	
+	
+ww:Agent rdf:type owl:Class ;
+	rdfs:label "Agent"@en ;
+	rdfs:comment "An individual or organisation that stands in some relationship to a work."@en ;
+	rdf:isDefinedBy <TBC/works> .
+
+ww:Format rdf:type owl:Class ;
+	rdfs:label "Format"@en ;
+	rdfs:comment "The format of a work: namely, whether it is a printed book, archive, image, moving image, etc."@en ;
+	rdf:isDefinedBy <TBC/works> .
+	
+ww:Location rdf:type owl:Class ;
+	rdfs:label "Location"@en ;
+	rdfs:comment "The place at which a work was published or otherwise created."@en ;
+	rdf:isDefinedBy <TBC/works> .
+	
+ww:Genre rdf:type owl:Class ;
+	rdfs:label "Genre"@en ;
+	rdfs:comment "The genre to which a work belongs: Genre being taken to mean a particular form carrying certain assumptions about content, authorial stance, etc.  Examples might include biography, article, obituary, satire, etc."@en ;
```

Comment:
Nit: either say “Examples might include” or “etc”, but I don’t think you need both.